flow/layers/backdrop_filter_layer.cc

```diff
@@ -9,6 +9,30 @@ namespace flutter {
 BackdropFilterLayer::BackdropFilterLayer(sk_sp<SkImageFilter> filter)
     : filter_(std::move(filter)) {}
 
+#ifdef FLUTTER_ENABLE_DIFF_CONTEXT
+
+void BackdropFilterLayer::Diff(DiffContext* context, const Layer* old_layer) {
+  DiffContext::AutoSubtreeRestore subtree(context);
+  auto* prev = static_cast<const BackdropFilterLayer*>(old_layer);
+  if (!context->IsSubtreeDirty()) {
+    FML_DCHECK(prev);
+    if (filter_ != prev->filter_) {
+      context->MarkSubtreeDirty(context->GetOldLayerPaintRegion(old_layer));
+    }
+  }
+
+  // Backdrop filter paints everywhere in cull rect
+  auto paint_bounds = filter_->computeFastBounds(context->GetCullRect());
+  context->AddPaintRegion(paint_bounds);
```

Comment:
Should this just be `AddPaintRegion(cull_rect)`?

Reply:
Fixed [here](https://github.com/flutter/engine/pull/21824/commits/d03308775de13e158ceb31e431fe1b97982aad11).